### PR TITLE
Update CoC to reflect the new CoC WG

### PR DESCRIPTION
**Pull request by nainar**
Changes points of contact to CoC WG

cc @estherkim @danielrozenberg @jpettitt. (Don't have the ability to assign reviewers.) Please let me know if the edits look good to you.

### CODE REVIEW

**Comment by mrjoro** [comment]
LGTM; thanks!

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ All members, committers and volunteers in this community are required to act acc
 
 ## Responsible and enforcing this code of conduct
 
-If you are being harassed, notice that someone else is being harassed, or have any other concerns, please contact us immediately: contact Malte Ubl (malte.ubl+coc@gmail.com) or Dima Voytenko dimav+coc@google.com. We are here to help you. Your reports will be taken seriously and not dismissed or argued with.
+If you are being harassed, notice that someone else is being harassed, or have any other concerns, please reach out to the [Code of Conduct Working Group](https://github.com/ampproject/wg-codeofconduct) [members](https://github.com/ampproject/wg-codeofconduct#members) immediately. We are here to help you. Your reports will be taken seriously and not dismissed or argued with.
```
Comment (danielrozenberg) [inline CODE_OF_CONDUCT.md]: Two links back-to-back can be confusing to readers. How about:
`please reach out to [members](https://github.com/ampproject/wg-codeofconduct#members) of the [Code of Conduct Working Group](https://github.com/ampproject/wg-codeofconduct) immediately.`

Comment (nainar) [reply CODE_OF_CONDUCT.md]: Done inline! Thanks!

**Comment by nainar** [comment]
@mrjoro could you please merge this? Thank you!

**Pull request merged**